docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
+- create number_of_floors_string, which will be appended to the details_of_system_selection string: `number_of_floors_string = ""`
+
+- get the climate zone category using the functions is_CZ_0_to_3a: `if is_CZ_0_to_3a():`
+	- set climate_zone_category to CZ_0_to_3a: `climate_zone_category = " CZ_0_to_3a"
+
+- else (the climate zone is 3b, 3c or 4 to 8): `else:`
+	- set climate_zone_category to CZ_3b_3c_or_4_to_8: `climate_zone_category = " CZ_3b_3c_or_4_to_8"
+
+- check if the building area type is residential: `if building_area_type == "RESIDENTIAL":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 1: `system_type = "SYS-2"`
+	- otherwise, it's  CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 2: `system_type = "SYS-1"
+
+
+- check if the building area type is Public Assembly: `if building_area_type == "PUBLIC_ASSEMBLY":`
+	- check if the space area is less than 120,000 ft2: `if area_of_BAT < 120000:`
+		- set area_of_BAT_string to "< 120,000": `area_of_BAT_string = " < 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 3: `system_type = "SYS-4"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 4: `system_type = "SYS-3"
+	- otherwise, the space area is greater than 120,000 ft2: `else:`
+		- set area_of_BAT_string to ">= 120,000": `area_of_BAT_string = " >= 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 12: `system_type = "SYS-13"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 13: `system_type = "SYS-12"
+
+
+- check if the building area type is heated-only storage: `if building_area_type == "HEATED-ONLY_STORAGE":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 9: `system_type = "SYS-10"`
+	- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 10: `system_type = "SYS-9"
+
+
+- check if the building area type is Retail: `if building_area_type == "RETAIL":`
+	- check if there are fewer than 3 floors in the building: `if number_of_floors < 3:`
+		- set number_of_floors_string to " 1 or 2 floors": `number_of_floors_string = " 1 or 2 floors"`
```

Comment:
i am unsure on how this should work. 

- need to look at building segment to get the number of floors. 
- Need to add number of floors above grade and number of floors below grade
- need to exclude floors devoted to parking. I am not sure if we will be able to tell within a building segment is a floor is devoted entirely to parking.

Reply:
again, suggest saving the number of floors as a number and not as a string. Comparisons using strings dont always work well.

Reply:
the `number_of_floors` is a variable that is passed into this function - it is calculated by the function (get_number_of_floors)[https://github.com/pnnl/ruleset-checking-tool/blob/RDS/KJW/function_get_number_of_floors/docs/ruleset_functions/get_number_of_floors.md]

Reply:
`number_of_floors_string` becomes part of the target string.  We don't need it to tell us the number of floors, just what contribution the number of floors had to the system selection (for example, the `number_of_floors_string` will always equal "" for Residential BAT) - again, the target string concept is a little messy, I'm open to other suggestions, but not sure the best way to achieve the same goal.

Reply:
@KarenWGard - Weili would be the best person to help with this. Maybe a target string is the best solution and Weili can clarify either ways. @weilixu